packages/fxa-settings/src/components/en.ftl

```diff
@@ -20,10 +20,14 @@
 # “Account” can be localized, “Firefox” must be treated as a brand.
 # This is used to refer to a user's account, e.g. "update your Firefox account ..."
 -product-firefox-account = Firefox account
-product-mozilla-vpn = Mozilla VPN
-product-pocket = Pocket
-product-firefox-monitor = Firefox Monitor
-product-firefox-relay = Firefox Relay
+-product-mozilla-vpn = Mozilla VPN
+-product-mozilla-mdn = Mozilla MDN
+-product-mozilla-hubs = Mozilla Hubs
+-product-pocket = Pocket
+-product-firefox-monitor = Firefox Monitor
+-product-firefox-relay = Firefox Relay
+-product-firefox-sync = Firefox Sync
+-product-firefox-addons = Firefox Add-ons
```

Comment:
This shouldn't be defined as a term, the message should be the term for Firefox + Add-ons

```
delete-account-product-firefox-addons = { -brand-firefox } Add-ons
```

Reply:
@flodolo _Just_ this one, or all of them should use `{ -brand-firefox }` or `{ -brand-mozilla }`?

Reply:
Only this one, terms should not reference other terms